### ISSUE TITLE
Integrate didyoumean.js to play a particular Pandora station

### ISSUE THREAD

**Issue by justinmiller61** [nfarina/homebridge-sonos#6]
I'm not sure if homebridge/homekit even has this ability, but I could see using didyoumean.js to find a matching Pandora station to play.

I haven't looked into whether or not this is even possible, but if it is, I'd be happy to fork and help out.

**Comment by NovaGL** [comment]
I don't believe this is possible as it's Siri doing the interpritations not Homebridge

**Comment by justinmiller61** [comment]
So Siri won't even send the command to homebridge?



—

**Comment by NovaGL** [comment]
No it won't work, as you would need to use the Pandora API to get matching stations.

**Comment by justinmiller61** [comment]
Riiiight. And what's wrong with doing that? I assumed as much. So assuming that's not a problem, does the entire transcription of what you said make it down to homebridge?



—

**Comment by NovaGL** [comment]
No, Siri only accepts predefined words. And won't accept others.

Eg Turn On Chill music would work using the HTTP plugin and [this] (https://github.com/jishi/node-sonos-http-api)

But then if you said switch to pandora station dance, it wouldn't work.

You can only use On or Off, it doesn't even accept volume.

So the best way I would do it is define several pandora stations you would like in your config.
```xml
 {
             "accessory": "Http",
            "name": "Hits",
            "on_url": "http://localhost:5005/Glenn/favorite/Todays Hits",
            "off_url": "http://localhost:5005/Glenn/pause",	
	        "service": "Switch",
       }
```
